debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorTask.java

```diff
@@ -53,17 +54,37 @@ public String version() {
         Configuration jdbcConfig = connectorConfig.getJdbcConfig();
         jdbcConnection = new OracleConnection(jdbcConfig, () -> getClass().getClassLoader());
 
+        validateRedoLogConfiguration();
+
         OracleValueConverters valueConverters = new OracleValueConverters(connectorConfig, jdbcConnection);
         TableNameCaseSensitivity tableNameCaseSensitivity = connectorConfig.getAdapter().getTableNameCaseSensitivity(jdbcConnection);
         this.schema = new OracleDatabaseSchema(connectorConfig, valueConverters, schemaNameAdjuster, topicSelector, tableNameCaseSensitivity);
         this.schema.initializeStorage();
         Offsets<OraclePartition, OracleOffsetContext> previousOffsets = getPreviousOffsets(new OraclePartition.Provider(connectorConfig),
                 connectorConfig.getAdapter().getOffsetContextLoader());
+
+        LOGGER.info("Closing connection before starting schema recovery");
+
+        try {
+            jdbcConnection.close();
+        }
+        catch (SQLException e) {
+            throw new DebeziumException(e);
+        }
+
```

Comment:
I meant to ask earlier @ashulin and I forgot, can you explain why the connection needs to be force closed?

Reply:
In fact, I have the same doubts. The code was not added during the internal modification of the company. I compared the mysql implementation before submitting the PR, and I copied it and kept it because I didn't know the impact of missing this code.

Reply:
Could you remove it as well as the code-block a few lines below that sets the `autoCommit` status?  I don't believe we need those changes and lets see how the code fairs with testing after those changes are implemented.

Reply:
I removed the code and passed the integration test.